src/main/java/com/gluonhq/substrate/SubstrateDispatcher.java

```diff
@@ -67,25 +65,21 @@ public static void main(String[] args) throws Exception {
         Triplet targetTriplet = targetProfile != null? new Triplet(Constants.Profile.valueOf(targetProfile.toUpperCase()))
                 :Triplet.fromCurrentOS();
 
-        ProjectConfiguration config = new ProjectConfiguration();
-        config.setGraalPath(graalVM);
-        config.setMainClassName(mainClass);
-        config.setAppName(appName);
-        config.setJavafxStaticSdkVersion(Constants.DEFAULT_JAVAFX_STATIC_SDK_VERSION);
-        config.setTarget(targetTriplet);
+        ProjectConfiguration publicConfig = new ProjectConfiguration(mainClass);
+        publicConfig.setGraalPath(graalVM);
+        publicConfig.setAppName(appName);
+        publicConfig.setJavafxStaticSdkVersion(Constants.DEFAULT_JAVAFX_STATIC_SDK_VERSION);
```

Comment:
remove this, so that the default is used automatically.
It makes no sense developers have to retrieve the default version and set that, this is what the configuration should hide.

Reply:
Fixed

Reply:
Fixed